### DISCUSSION TITLE
Gendern

### DISCUSSION THREAD

**Discussion by SpotlightForBugs** [Bootstrap-Academy/Bootstrap-Academy#40] [Q&A] [answered]
Inwiefern sollte Gendern eingesetzt werden? 

(Keine Grundsatzdiskussion, bitte einfach die Art und Weise, wie ihr Gendern wollt oder Nein)

**Comment by SpotlightForBugs** [comment]
Möglichkeit: so wie Apple (Benutzer:innen)

**Comment by SpotlightForBugs** [comment]
Gar nicht

**Comment by SpotlightForBugs** [comment]
Ja, mir egal wie

**Comment by Defelo** [comment]
Hast du irgendwelche Beispiele, wo das relevant wäre?

**Comment by TheMorpheus407** [comment, marked as answer]
FInde ich derzeit nicht so wichtig, dass wir uns direkt drum kümmern sollten. Wenn dann würde ich gerne die Option geben, dass man sein Pronomen setzen kann und anschließend alles passend angezeigt wird, aber dann wird das ein größeres Feature^^
